charts/promtail/values.yaml

```diff
@@ -4,20 +4,19 @@ nameOverride: null
 # -- Overrides the chart's computed fullname
 fullnameOverride: null
 
-initContainer:
-  # -- Specifies whether the init container for setting inotify max user instances is to be enabled
-  enabled: false
-  image:
-    # -- The Docker registry for the init container
-    registry: docker.io
-    # -- Docker image repository for the init container
-    repository: busybox
-    # -- Docker tag for the init container
-    tag: 1.33
+initContainers:
```

Comment:
I think it would be better to provide an empty list and provide a commented example. By default we don't enable init containers.